resources/assets/styles/components/button.scss

```diff
@@ -22,6 +22,22 @@
         map-get($theme-colors, "primary"),
         map-get($theme-colors, "light")
       );
+
```

Comment:
I didn't mean to include this scss stuff. 

It does fix the block preview on our example gutenberg button but I'll open that as  a separate thing.

Reply:
Reverted.